lib/guard/standardrb/version.rb

```diff
@@ -1,5 +1,5 @@
 module Guard
   module StandardrbVersion
-    VERSION = "0.1.1".freeze
+    VERSION = "0.2.0".freeze
```

Comment:
Leave it up to the gem maintainers to bump version numbers themselves.

Reply:
yeah I am the gem maintainer haha. I though because I am changing the input the method I should make it a minor bump and not a patch version